bridge_adaptivity/module/tests/test_models.py

```diff
@@ -3,13 +3,15 @@
 from module import models
 from module.engines.engine_mock import EngineMock
 from module.engines.engine_vpal import EngineVPAL
-from module.models import Engine
+from module.models import Engine, GradingPolicy
+from module.policies.policy_points_earned import PointsEarnedGradingPolicy
+from module.policies.policy_trials_count import TrialsCountGradingPolicy
 
 
 class TestEngineUtilityFunction(TestCase):
     def test__discover_engines(self):
         """Test _discover_engines function."""
-        found_engines = models._discover_engines()
+        found_engines = models._discover_applicable_modules(folder_name='engines', file_startswith='engine_')
```

Comment:
engines and engine_ are default values and could be avoided, but it is good to have for clarity.

Reply:
that's what I wrote here.